exchange-v2/test/v2/ExchangeV2.rarible.test.js

```diff
@@ -1037,9 +1037,9 @@ contract("ExchangeV2, sellerFee + buyerFee =  6%,", accounts => {
       const left1 = Order(seller, Asset(ERC1155, enc(erc1155_v2.address, erc1155TokenId1), 200), ZERO, Asset(ETH, "0x", 600), 1, 0, 0, ORDER_DATA_V2, encDataLeft);
       const right1 = Order(buyer1, Asset(ETH, "0x", 300), ZERO, Asset(ERC1155, enc(erc1155_v2.address, erc1155TokenId1), 100), 1, 0, 0, ORDER_DATA_V2, encDataRight);
   
-      await verifyBalanceChange(seller, -261, async () =>
+      await verifyBalanceChange(seller, -241, async () =>
         verifyBalanceChange(buyer1, 309, async () =>
-          verifyBalanceChange(accounts[5], -30, async () =>
+          verifyBalanceChange(accounts[5], -50, async () =>
```

Comment:
This is the only test I had to change the expected values.
This is because of partial fill, the origin fees are now 100% applied